streamz/tests/test_dask.py

```diff
@@ -50,6 +51,25 @@ def add_to_dict(d):
         assert item["i"] == i
 
 
+@gen_cluster(client=True)
+def test_non_unique_emit(c, s, a, b):
+    """Regression for https://github.com/python-streamz/streams/issues/397
+
+    Non-unique stream entries still need to each be processed.
+    """
+    source = Stream(asynchronous=True)
+    futures = source.scatter().map(lambda x: random.random())
+    L = futures.gather().sink_to_list()
+
+    for _ in range(3):
+        # Emit non-unique values
+        yield source.emit(0)
+
+    print(L)
```

Comment:
leftover print

Reply:
Fixed. I find those kind of handy when looking at pytest output, esp. when the printed variable doesn't show up in an assertion statement (it can't here because the use of `random`). Regardless, removed for streamz convention purposes.